src/intl/en/page-eth2-index.json

```diff
@@ -40,13 +40,13 @@
   "page-eth2-diagram-validators": "More on validators",
   "page-eth2-dive": "Dive into the vision",
   "page-eth2-dive-desc": "How are we going to make Ethereum more scalable, secure, and sustainable? All while keeping Ethereum's core ethos of decentralization.",
-  "page-eth2-docking": "The docking",
-  "page-eth2-docking-answer-1": "The docking is when mainnet turns into a shard. This will come after a successful implementation of shard chains.",
-  "page-eth2-docking-btn": "More on the docking",
-  "page-eth2-docking-desc": "Mainnet Ethereum will need to “dock” or “merge” with the Beacon Chain at some point. This will enable staking for the entire network and signal the end of energy-intensive mining.",
-  "page-eth2-docking-estimate": "Estimate: 2022",
+  "page-eth2-docking": "The merge",
+  "page-eth2-docking-answer-1": "The merge is when the mainnet begins using the Beacon Chain for consensus, and proof-of-work is turned off.",
+  "page-eth2-docking-btn": "More on the merge",
+  "page-eth2-docking-desc": "Mainnet Ethereum will need to “merge” with the Beacon Chain at some point. This will enable staking for the entire network and signal the end of energy-intensive mining.",
+  "page-eth2-docking-estimate": "Estimate: 2021",
   "page-eth2-docking-mainnet": "What's mainnet?",
-  "page-eth2-docking-mainnet-eth2": "Docking mainnet into Eth2",
+  "page-eth2-docking-mainnet-eth2": "Merging mainnet into Eth2",
```

Comment:
Maybe merging mainnet and the Beacon Chain would be better.. We want to move away from Eth2 being a thing

Reply:
Agreed.

Reply:
Has this been changed? can't work out where this string is meant to be haha?